### PR TITLE
fix: Timestamp update

### PR DESCRIPTION
**Pull request by kevjumba**
**What this PR does / why we need it**:
Updates data source api by replacing `event_timestamp_column` with `timestamp_field` param.
**Which issue(s) this PR fixes**:

Fixes #

### CODE REVIEW

**Comment by kevjumba** [comment]
I just want to run integration tests to find any major bugs that need to be fixed.

**Comment by achals** [comment]
You may need to change https://github.com/feast-dev/feast/blob/master/java/serving/src/test/resources/docker-compose/feast10/definitions.py as well.

**Comment by kevjumba** [comment]
> https://github.com/feast-dev/feast/blob/master/java/serving/src/test/resources/docker-compose/feast10/definitions.py

Hm I believe my pr does update the param.

File: sdk/python/feast/data_source.py [hunk 1]
```diff
@@ -384,14 +396,15 @@ def __eq__(self, other):
...
+            event_timestamp_column=data_source.timestamp_field,
             field_mapping=dict(data_source.field_mapping),
             bootstrap_servers=data_source.kafka_options.bootstrap_servers,
             message_format=StreamFormat.from_proto(
                 data_source.kafka_options.message_format
             ),
             topic=data_source.kafka_options.topic,
-            event_timestamp_column=data_source.event_timestamp_column,
-            created_timestamp_column=data_source.created_timestamp_column,
+            created_timestamp_column=data_source.timestamp_field,
```
Comment (achals) [inline sdk/python/feast/data_source.py:406]: is this right? Shouldn't this be `data_source.created_timestamp_column`?

Comment (kevjumba) [reply sdk/python/feast/data_source.py:406]: Ah good catch, fixed.

**Comment by kevjumba** [comment]
@achals Could you stamp this?

**Pull request merged**